src/Features/CSharp/Portable/xlf/CSharpFeaturesResources.cs.xlf

```diff
@@ -67,11 +67,6 @@
         <target state="translated">Automatický výběr je zakázaný kvůli možné deklaraci proměnné rozsahu.</target>
         <note />
       </trans-unit>
-      <trans-unit id="Remove_Unnecessary_Cast">
```

Comment:
already had this loc string in the common features layer.  So i removed it from both the C# and VB layers.  We should def have an analyzer that catches this.